### ISSUE TITLE
2.0.7 compiled but fails on make install 

### ISSUE THREAD

**Issue by tseemann** [TGAC/KAT#28]
I think you are trying ot change to a folder called `/deps` because `$(DESTDIR)` is empty ?

```
make  install-data-hook
make[3]: Entering directory '/home/tseemann/git/KAT'
cd /deps; \
cat deps/seqan-library-2.0.0.tar.gz | gunzip |  -H gnu --list > uninstall_manifest.txt; \
cat deps/seqan-library-2.0.0.tar.gz | gunzip |  -H gnu --no-same-owner --extract; \
rm --force deps/seqan-library-2.0.0.tar.gz; \
cat uninstall_manifest.txt | sed --expression='s/^\|$/"/g' | xargs chmod a=rX,u+w
/bin/sh: line 0: cd: /deps: No such file or directory
/bin/sh: line 1: -H: command not found
/bin/sh: line 2: -H: command not found
chmod: missing operand after ‘a=rX,u+w’
Try 'chmod --help' for more information.
Makefile:941: recipe for target 'install-data-hook' failed
make[3]: *** [install-data-hook] Error 123
make[3]: Leaving directory '/home/tseemann/git/KAT'
Makefile:867: recipe for target 'install-data-am' failed
make[2]: *** [install-data-am] Error 2
make[2]: Leaving directory '/home/tseemann/git/KAT'
Makefile:821: recipe for target 'install-am' failed
make[1]: *** [install-am] Error 2
make[1]: Leaving directory '/home/tseemann/git/KAT'
Makefile:520: recipe for target 'install-recursive' failed
make: *** [install-recursive] Error 1
```

**Comment by maplesond** [comment]
Ahh, ok I think I see where this is going wrong.  I normally only test the `make install` when I've giving it a specific prefix.  In this case when there's no prefix provided it tries to cd into `/deps` which doesn't exist.  I'll implement a fix for this and apply to 2.0.8 and 2.1.0.

**Comment by maplesond** [comment]
I've reminded myself why I put this in in the first place.  Basically, I was trying to avoid treating seqan as a separate sub project with it's own Makefile.  The cleanest solution is to simply so what I should have done originally and create a seqan Makefile with all headers under the automake macro `noinst_HEADERS`.  This produces the correct behaviour for both `make install` and `make dist`.

**Comment by maplesond** [comment]
Fixed in V2.0.8.
